### ISSUE TITLE
Current State of Apollo Client and Server tutorials

### ISSUE THREAD

**Issue by KKS1** [howtographql/howtographql#1334]
Hi team,

First of all thanks a bunch for giving this tremendous jewel to dev community.
We had a quick question, on the current state of Apollo Client and Server tutorials in comparison to whats there on https://www.apollographql.com/ that is
https://odyssey.apollographql.com/?_gl=1*1fjmv50*_ga*MzQ5MjIyNjYzLjE2MjQ1NjIxNjA.*_ga_0BGG5V2W2K*MTYyNDU2MjE2MC4xLjAuMTYyNDU2MjE2MC4w.

Trying to access, which of these (howtographql vs apollographql) are representing the current/ updated state of Apollo GraphQL ecosystem that should be used for a new application, when starting from scratch.

Thank you!

**Comment by TasinIshmam** [comment]
Hi @KKS1,

I haven't had the opportunity to go in-depth into the apollographql tutorial that you have mentioned. However, I took a glance, and my impression is that it does a solid job of teaching the core topics. While I don't have a concrete answer for you, I think both tutorials are quite viable. Both tutorials use a relatively recent version of Apollo Server and Client, so you shouldn't have any issues with either. 

I could be making certain undue assumptions here, but since the apollographql content is maintained by the Apollo team, it might cover Apollo-specific tooling in more depth. If that's your intention, I'd definitely suggest giving those tutorials a try. 

On the other hand, if you're looking for a good general introduction to GraphQL and the general ecosystem, I hope you'll find howtographql a valuable starting resource.

I hope I was able to provide some insight!
